README.md

```diff
@@ -26,8 +27,8 @@ When not explicitly set, files are placed under [![Eclipse license](https://img.
 
 #### Version:
 
-* ``2.0.0`` Stable OH2 version
-* ``2.1.0-SNAPSHOT`` Current OH2 snapshot version
+* ``2.0.0`` Stable Openhab version
```

Comment:
@legacycode Just as a reminder: It is spelled "openHAB", not "Openhab", so please make sure to do this right in all your work - thanks!